app/services/movie_service.rb

```diff
@@ -1,7 +1,7 @@
 class MovieService 
 
-  def initialize(url = '')
-    @url = url
+  def initialize(movie_id = '')
+    @movie_id = movie_id
   end
```

Comment:
This change to movie_id better reflects how this is currently functioning, but the idea was to allow us to pass in a variety of query param type data regardless of which api endpoint we wanted to hit. I think this is an improvement, but there is still room for refactoring this class.